### PR TITLE
Add basic development docs

### PR DESCRIPTION
**Pull request by openjck**

### CODE REVIEW

**Comment by spasovski** [comment]
Looks good from my end. `npm install` failed without the previous PR but is fine now.

**Comment by openjck** [comment]
I'll add the `gulp build` post-install task back. That's fine now that Gulp is all set up.

**Comment by openjck** [comment]
Updated

**Comment by lonnen** [comment]
can these be integrated into a README?

**Comment by openjck** [comment]
> can these be integrated into a README?

Will do!

File: README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+6. In another terminal, set up and run the database
+    1. Install postgres
+        * Mac: `brew install postgres`
+    2. Create a new user with name *distributionviewer* and password *distributionviewer*
+        * Mac: `createuser --pwprompt distributionviewer`
+    3. Create a new database named *distributionviewer* that the
+       *distributionviewer* user has read/write access to
+        * Mac: `createdb -Odistributionviewer -Eutf8 distributionviewer`
+    4. Run postgres
+        * Mac: `postgres -D /usr/local/var/postgres`
```
Comment (robhudson) [inline README.md]: To create the user and database this actually has to happen first. For myself I ran `brew services start postgresql` so it runs in the background. But this is probably a personal preference type of thing.

Comment (openjck) [inline README.md]: Ah, I like that approach. Didn't know `brew` could do that. I'll move this up, too.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+1. `echo DEBUG=True >> .env`
+2. `virtualenv env`
+3. `source env/bin/activate`
+4. `pip install -r requirements.txt`
+5. `npm install`
+6. In another terminal, install and run the database
+    1. Install postgres
+        * Mac: `brew install postgres`
+    2. Run postgres
+        * Mac: `brew services start postgresql`
```
Comment (robhudson) [inline README.md]: This is also found in the `brew info postgresql` documentation. If we wanted to suggest reading that?

**Comment by robhudson** [comment]
r+wc

**Comment by openjck** [comment]
Updated 🍭

**Pull request merged**